From 2df49a05e0a93a968358484eecce7b6d74f3caad Mon Sep 17 00:00:00 2001
From: Ji Luo <ji.luo@nxp.com>
Date: Fri, 8 Sep 2023 12:21:56 +0800
Subject: [PATCH] MATTER-1352-2 Add se_version.h

This header file would be generated by the Makefile in
imx_secure_enclave repo, we hard-code it here because
we can't generate it after integrating it to matter.

Change-Id: I94053d9231c7a3323067ffb59b15f516fae69cf1
Signed-off-by: Ji Luo <ji.luo@nxp.com>
---
 include/hsm/internal/se_version.h | 10 ++++++++++
 1 file changed, 10 insertions(+)
 create mode 100644 include/hsm/internal/se_version.h

diff --git a/include/hsm/internal/se_version.h b/include/hsm/internal/se_version.h
new file mode 100644
index 0000000..23bc49d
--- /dev/null
+++ b/include/hsm/internal/se_version.h
@@ -0,0 +1,10 @@
+#ifndef SE_VERSION_H
+#define SE_VERSION_H
+#define LIB_COMMIT_ID "9179ef03560d9639ec35948ac4169d5e4364dc4c"
+#define LIB_NEWNESS_VER 1
+#define LIB_MAJOR_VER 1
+#define LIB_MINOR_VER 0
+#define NVM_NEWNESS_VER 1
+#define NVM_MAJOR_VER 1
+#define NVM_MINOR_VER 0
+#endif
-- 
2.25.1

